addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -305,9 +310,16 @@ export class SearchBar extends Component {
         const options = await this.orm.call(relation, "name_search", [], {
```

Comment:
I think you should extract the limit in a const (possible problems with async code here)